lib/tasks/ffmpeg.rake

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+namespace :ffmpeg do
+  task queue_version_checks: :environment do
+    ContentLength.find_each do |cl|
+      cl.delay(queue: :content_lengths).check_ffmpeg_version
```

Comment:
Doesn't using the backlog queue make more sense?

Reply:
We need to verify a whether the content_lengths can be used quicker than we need the content_lengths themselves, so I would give these two tasks a different priority